From: Luotao Fu <l.fu@pengutronix.de>
Subject: fix include of tslib.c

tslib.c is missing an include of string.h, which causes to compiling error: 
"implicit declaration of function 'strcmp'" This one adds the missing include.
This is fixed in newer versions.

Signed-off-by: Luotao Fu <l.fu@pengutronix.de
---
 inputdrivers/tslib/tslib.c |    1 +
 1 file changed, 1 insertion(+)

Index: inputdrivers/tslib/tslib.c
===================================================================
--- inputdrivers/tslib/tslib.c.orig
+++ inputdrivers/tslib/tslib.c
@@ -32,6 +32,7 @@
 
 #include <stdlib.h>
 #include <stdio.h>
+#include <string.h>
 
 #include <tslib.h>
 
